src/main/java/com/volmit/iris/core/decrees/DecIris.java

```diff
@@ -18,20 +18,26 @@
 
 package com.volmit.iris.core.decrees;
 
+import com.volmit.iris.Iris;
 import com.volmit.iris.util.decree.DecreeExecutor;
 import com.volmit.iris.util.decree.annotations.Decree;
 import com.volmit.iris.util.decree.annotations.Param;
 
 @Decree(name = "irisd", aliases = {"ird"}, description = "Basic Command")
-public class CMDIris implements DecreeExecutor
+public class DecIris implements DecreeExecutor
 {
-    private CMDIrisStudio studio;
+    private DecIrisStudio studio;
 
-    @Decree(description = "Ping self", aliases = "p")
+    @Decree(description = "Send a message to yourself", aliases = "p")
```

Comment:
This is a test command, it can actually be removed

Reply:
Two